README.md

```diff
@@ -1,13 +1,237 @@
+Have you ever struggled with different config values across local, CI,
+and various cluster environments? If no, did you ever forget to update the
+cluster config after updating the local config? Have you ever been worried about
+the secrecy of your DB credentials and API keys taking the `.env` file approach?
+Ever leaked confidential data by pushing secrets in your `.env` file to the git remote?
+
+Do you agree that just `dotenv` is not enough? **`strong-config` is here to help!**
+
+- Structure configs hierarchically by writing JSON or YAML.
+- Manage different configs for different environments, e.g. different clusters
+- Protect your secrets by applying strong cryptography. Encrypt secrets at rest
+  and automatically decrypt them at runtime by integrating a key management
+  service (KMS), e.g. [AWS KMS](https://aws.amazon.com/kms/) or
+  [Google Cloud KMS](https://cloud.google.com/kms/).
+- Enforce arbitrary permission models provided by your KMS. Decide who can encrypt
+  and decrypt configs for which cluster environments.
+- Optionally validate them against a custom schema using
+  [`json-schema`](https://json-schema.org/). Also in CI.
+- Safeguard your configs by using git hooks. Make sure the configs are both
+  valid and encrypted before committing and pushing them.
+- Autogenerate types based on your schema. These types can be imported in your
+  sources to write safer code (Typescript only for now).
+
+These benefits help you to catch inconsistencies, exposed secrets, and invalid
+configs as early as possible in the development process. To do so, you will use
+widely adopted tools you are probably already familiar with: JSON (or YAML),
+your cloud provider, and yet another npm package. Try it out!
+
+## Example
+
+Here is how an encrypted config file containing secrets looks like:
+
+```yml
+logger:
+  level: DEBUG
+auth:
+  url: my-auth-url
+  myApiKeySecret: ENC[AES256_GCM,data:aeQ+hlVIah7WyJoVR/Jbkb6GLH7ihsV0D81+U++pkiWD0zeoRL/Oe9Q3Tz6j/TNvKKVDnohIMyw3UVjELOuSY+A==,iv:nVRZWogV4B7o=,tag:KrE2jssfP4uCvqq+pc/JyQ==,type:str]
+sops:
+  gcp_kms:
+  -   resource_id: projects/my-project/locations/europe-west3/keyRings/my-project-key-ring/cryptoKeys/my-strong-config-key
+      created_at: '2019-11-07T10:11:12Z'
+      enc: AiAAmdAgjOdw1XdV2MsDpvmZ4Deo867hmcX2B3DNhe2BCF2axuZ18hJJFK9oBlE1BrD70djwqi+L8T+NRNVnGUP+1//w8cJATAfJ8W/cQZFcdFTqjezC+VYv9xYI8i1bRna4xfFo/INIJtFDR38ZH1nrQg==
+  lastmodified: '2019-11-07T10:11:12Z'
+  mac: ENC[AES256_GCM,data:ABcd1EF2gh3IJKl4MNOpQr5stuvWXYz6sBCDEfGhIjK=,iv:A1AaAAAaa111a1Aa111AA/aaaAaaAAaa+aAaAaAAAaA=,tag:AAaaA1a1aaaAa/aa11AaaA==,type:str]
+  encrypted_suffix: Secret
+  version: 3.4.0
+```
+
+The field `sops` contains a set of metadata provided context information and
+important details to decrypt this config at runtime. Never delete this field
+or make changes to the encrypted file without using `@strong-config` - it will
```

Comment:
Jokes are fine and it **is** a problem to delete this value, but we don't want to scare people off too much here.

Reply:
yeah we dont want to scare people. done